README.md

```diff
@@ -80,6 +81,12 @@ show how sources generated from the schema could be linked in the worker
 binary. See `schema/CMakeLists.txt` which creates a library with all generated
 sources.
 
+You can see (and edit) the content of the snapshot in text format by running a command to convert it:
+
+```
+spatial project history snapshot convert --input <path> --input-format binary --output <path> --output-format text
```

Comment:
Do you want to use consistent `--foo=bar` style here too?

Reply:
Changing the style yep. Would opt to leave out the quotes around <path> for the sake of clarity.